pyfuncol/tests/test_set.py

```diff
@@ -1,35 +1,48 @@
+from operator import ne
 import pyfuncol
 
 s = {1, 2, 3}
 st = frozenset(s)
 
 
 def test_map():
+    s = {1, 2, 3}
```

Comment:
@samuelchassot why do you need to do this for each one?

Reply:
I modified the sets in one test and it made the others failed. This is bad, tests should be decoupled and as `beforeEach` isn't really encouraged by pytest, I made it the old way :)

Reply:
I see! I would still either use before each with fixtures, or find a way to not modify the sets, it's quite a lot of code added for basically nothing

Reply:
I quite agree about the amount of code, however I do not really like the fact that tests act on the same set so I would either go for the beforeEach (which is not really advised in pytest by the developers apparently) or as is. IMHO it is not really a problem to have code duplicates in unit tests, it can even help with the readability (specially when it is 2-3 lines)

Reply:
Ok!